WhaleSpotting/Data/ApplicationDbContext.cs

```diff
@@ -7,6 +7,8 @@ public class ApplicationDbContext : IdentityDbContext
 {
     public DbSet<Sighting>? Sightings { get; set; }
 
+    public DbSet<Species>? Species { get; set; }
+
```

Comment:
Prefer the `DbSet`s to all be next to each other without blank lines
```suggestion
    public DbSet<Species>? Species { get; set; }

```